src/assets/styles/element-ui.scss

```
@@ -119,90 +119,142 @@
 .el-dialog,
 .el-message-box {
   border-radius: 22px;
-  overflow: hidden;
+  // Element Plus 新结构：padding 在 .el-dialog 外层，header/body 不再各加左右 padding
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)